### PR TITLE
VAPI-922 Update callback docs to be more specific

### PR DESCRIPTION
**Pull request by tmoney-bw**
## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

File: site/src/pages/changelog.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ slug: /changelog
 
 | Date | Notes |
 |--|--|
+| November 17, 2021 | Update application callback docs to be clear about what is sent when|
```
Comment (mhossomi) [inline site/src/pages/changelog.md:10]: _[OCD intensifies]_
```suggestion
| November 17, 2021 | Update application callback docs to be clear about what is sent when |
```

**Review by mhossomi — APPROVED** [review]

**Review by mhossomi — APPROVED** [review]

File: site/docs/voice/quickStart.mdx [hunk 2]
```diff
@@ -32,8 +32,8 @@ Bandwidth will expect BXML in the HTTP response from either of these URLs to con
 These URLs can be the same or different, depending on if you would like inbound and outbound calls to be handled differently.
 
 
-### Call Status Callbacks
-Bandwidth will also send call status updates to the configured `CallStatusCallbackUrl`. These webhooks are useful for error checking and include events such as _rejected_ and _hung up_.
+### Disconnect Callbacks
```
Comment (CatexB) [inline site/docs/voice/quickStart.mdx:35]: Should we have a paragraph for initiate?

Comment (tmoney-bw) [reply site/docs/voice/quickStart.mdx:35]: We kind of do above this but I could be convinced to roll both of these into a single paragraph

Comment (CatexB) [reply site/docs/voice/quickStart.mdx:35]: Yeah a single paragraph would be fine, imo

**Review by CatexB — APPROVED** [review]

File: site/docs/voice/quickStart.mdx [hunk 3]
```diff
@@ -26,17 +26,17 @@ With an account and application created, we can get started using the Voice API.
...
 ## Configuring Callback URLs
 The voice API can be configured to use multiple different callback URLs to direct traffic and receive input from phone calls.
-Every application does require a call initiated callback URL, as this is the URL Bandwidth sends an `initiated` webhook to when an inbound call to a Bandwidth number is made.
-The other most widely used callback URL is the `answerUrl` in the POST request made to start an outbound call from a Bandwidth number.
-Bandwidth will expect BXML in the HTTP response from either of these URLs to continue the call using BXML verbs.
-These URLs can be the same or different, depending on if you would like inbound and outbound calls to be handled differently.
+Every application requires a `CallInitiatedCallbackUrl`, as this is the URL Bandwidth sends an [initiated webhook](./webhooks/initiate.mdx) to when an inbound call to a Bandwidth number is made,
+and a BXML response is expected to continue the call.
 
+For inbound calls, Bandwidth will also send [disconnect webhooks](./webhooks/disconnect.mdx) to the configured `CallStatusCallbackUrl`. This webhook is useful for error checking and include the cause such as _rejected_ and _hung up_.
```
Comment (n-miles) [inline site/docs/voice/quickStart.mdx:32]:
```suggestion
For inbound calls, Bandwidth will also send [disconnect webhooks](./webhooks/disconnect.mdx) to the configured `CallStatusCallbackUrl`. This webhook is useful for error checking and includes the cause of the disconnect, for example _rejected_ or _hung up_.
```

**Review by n-miles — DISMISSED** [review]

**Review by n-miles — APPROVED** [review]

**Review by CatexB — APPROVED** [review]

**Pull request merged**